charts/argo-cd/templates/NOTES.txt

```diff
@@ -52,6 +52,18 @@ DEPRECATED option configs.gpgKeys - Use config.gpg.keys
 {{- if .Values.configs.gpgKeysAnnotations }}
 DEPRECATED option configs.gpgKeysAnnotations - Use config.gpg.annotations
 {{- end }}
+{{- if hasKey .Values "createAggregateRoles" }}
+DEPRECATED option createAggregateRoles - Use global.rbac.aggregatedRoles
```

Comment:
Hey @pdrastil 

I am seeing this deprecation warning when upgrading my chart but the change is not done in the default values.yaml, just checking if this change is a mistake or there is future plan to move to global setting?

Reply:
Hi @faizsiddiqui - you can ignore this. It was originally placed into global with other cluster role rbac. This is leftover that shouldn't be there. Thanks for a find. I'll update the docs / notes.

Reply:
Resolved via https://github.com/argoproj/argo-helm/pull/1830